tests/test_class_basics.rs

```diff
@@ -244,3 +244,47 @@ fn panic_unsendable_base() {
 fn panic_unsendable_child() {
     test_unsendable::<UnsendableChild>().unwrap();
 }
+
+fn get_length(obj: &PyAny) -> PyResult<usize> {
+    let length = obj.len()?;
+
+    Ok(length)
+}
+
+#[pyclass]
+struct ClassWithFromPyWithMethods {}
+
+#[pymethods]
+impl ClassWithFromPyWithMethods {
+    fn instance_method(&self, #[pyo3(from_py_with = "get_length")] argument: usize) -> usize {
```

Comment:
Just curious; what happens if you try `"PyAny::len" instead of `"get_length"` in the macro? I guess it'll work? 🤔

Reply:
Hmm, I think it should work? I'm going to check it to be sure

Reply:
Yep, it works! Do you think we should use it instead?

Reply:
Might be nice to use one of each? :)

Reply:
done!